### PR TITLE
[Bank] Add PSECU

### PR DESCRIPTION
**Pull request by cicku**

### CODE REVIEW

File: data/brands/amenity/bank.json [hunk 1]
```diff
@@ -8078,6 +8078,18 @@
         "short_name": "PenFed"
       }
     },
+    {
+      "displayName": "Pennsylvania State Employees Credit Union",
+      "locationSet": {"include": ["us"]},
```
Comment (arch0345) [inline data/brands/amenity/bank.json:8083]:
```suggestion
      "locationSet": {"include": ["us-pa.geojson"]},
```
I'm assuming PSECU only has branches in Pennsylvania

File: data/brands/amenity/bank.json [hunk 2]
```diff
@@ -8078,6 +8078,18 @@
...
+      "tags": {
+        "alt_name": "Pennsylvania State Employees Credit Union",
+        "amenity": "bank",
+        "brand": "PSECU",
+        "brand:wikidata": "Q7120626",
+        "name": "Pennsylvania State Employees Credit Union",
+        "short_name": "PSECU"
```
Comment (arch0345) [inline data/brands/amenity/bank.json:8090]:
```suggestion
        "name": "PSECU"
```
`alt_name`/`name`/`short_name` should be different from one another. Based on their website it looks like they brand themselves as just 'PSECU' so I've suggested to change the name to this.

**Comment by okongu** [comment]
>

**Comment by okongu** [comment]
downloadfile.PDF

>

**Pull request merged**